Sources/DataPipeline/CustomerIO.swift

```diff
@@ -0,0 +1,28 @@
+import CioInternalCommon
+
+public extension CustomerIO {
+    /**
+     Initialize the shared `instance` of `CustomerIO`.
+     Call this function when your app launches, before using `CustomerIO.instance`.
+     */
+    @available(iOSApplicationExtension, unavailable)
+    static func initialize(
+        writeKey: String,
+        logLevel: CioLogLevel = .error,
```

Comment:
I was not expecting log level to be a parameter but instead be set via config object? 

I think consistency of all optional parameters, such as log level, being set inside of a config object.

Reply:
Thanks for that link. I didn't see that conversation. That does indeed answer my question. 

I am not the biggest fan of the public-API after that decision, but I understand why that decision was made. I would like to see more consistency in the public API of our SDK but truly making it consistent goes beyond the scope of the CDP SDK launch. So, I am OK with the public API we have here for now.